extensions/blocks/podcast-player/templates/podcast-header-title.php

```diff
@@ -17,26 +17,39 @@
  * @var array  $primary_colors
  */
 
-if ( ! isset( $title ) && empty( $track ) && ! isset( $track['title'] ) ) {
+if ( ! isset( $title ) && empty( $track['title'] ) ) {
 	return;
 }
+
+$track_link = empty( $track['link'] ) ? $track['src'] : $track['link'];
 ?>
 
 <h2 id="<?php echo esc_attr( $player_id ); ?>__title" class="jetpack-podcast-player__title">
-	<?php if ( ! empty( $track ) && isset( $track['title'] ) ) : ?>
-		<span
-			class="jetpack-podcast-player__current-track-title <?php echo esc_attr( $primary_colors['class'] ); ?>"
-			<?php echo isset( $primary_colors['style'] ) ? 'style="' . esc_attr( $primary_colors['style'] ) . '"' : ''; ?>
-		>
-			<?php echo esc_html( $track['title'] ); ?>
-		</span>
-	<?php endif; ?>
+	<span
+		class="jetpack-podcast-player__current-track-title <?php echo esc_attr( $primary_colors['class'] ); ?>"
+		<?php echo isset( $primary_colors['style'] ) ? 'style="' . esc_attr( $primary_colors['style'] ) . '"' : ''; ?>
+	>
+		<?php
+		echo esc_html( $track['title'] );
+		if ( ! empty( $track_link ) ) :
+			// Prevent whitespace between title and link to cause a jump when JS kicks in.
+			?>
+			<a
```

Comment:
@jeherve Is there a way to keep the auto-formatter from adding a line break between the closing PHP delimiter and the start of the anchor element?

Reply:
You'd have to skip the pre-commit hook I think.

Another option, maybe, would be to switch to a big `printf`, or `heredoc`?

Reply:
`--no-verify` did it. Thanks! Should be ready for another review now.